e2e/testdata/fn-eval/out-of-place-dir-exists-error-sink/.expected/exec.sh

```diff
@@ -0,0 +1,21 @@
+#! /bin/bash
```

Comment:
these tests should really be in `testdata/fn-sink` as it's testing `sink` not `eval`. Same with tests for `source`.

Reply:
Moved sink tests. Source and sink both go together. Not sure how to segregate them. Moved sink tests for now. We can revisit and group all these tests in a separate PR.